### PR TITLE
Various - Conform isNil checks across the codebase

### PR DESCRIPTION
**Pull request by Whigital**
**When merged this pull request will:**
- Update `isNil` checks from `!(isNil ....` -> `(!isNil ....` where applicable.
- Update `isNil` arg from `....isNil _arg)` -> `....isNil "_arg")` where applicable.
- Address https://github.com/acemod/ACE3/issues/7402 through https://github.com/acemod/ACE3/commit/2688cdd95f421d60aeabd80d82e203eb4dac708b

### CODE REVIEW

File: addons/common/functions/fnc_cbaSettings.sqf [hunk 1]
```diff
@@ -73,7 +73,7 @@ GVAR(settingsMovedToSQF) = [];
     #ifdef DEBUG_MODE_FULL
     INFO_1("checking settingsMovedToSQF [%1]",count GVAR(settingsMovedToSQF));
     {
-        if (isNil _x) then { WARNING_1("setting [%1] NOT moved to sqf",_x); };
+        if (isNil "_x") then { WARNING_1("setting [%1] NOT moved to sqf",_x); };
```
Comment (jonpas) [inline addons/common/functions/fnc_cbaSettings.sqf]: Are you sure `_x` isn't already a string?

File: addons/common/functions/fnc_cbaSettings.sqf [hunk 2]
```diff
@@ -107,7 +107,7 @@ for "_index" from 0 to (_countOptions - 1) do {
     if ((toLower _settingName) in GVAR(cbaSettings_forcedSettings)) then {
         WARNING_1("Setting [%1] - Already Forced - ignoring missionConfig",_varName);
     } else {
-        if ((isNil _settingName) && {(getNumber (_settingsConfig >> _settingName >> "movedToSQF")) == 0}) then {
+        if ((isNil "_settingName") && {(getNumber (_settingsConfig >> _settingName >> "movedToSQF")) == 0}) then {
```
Comment (jonpas) [inline addons/common/functions/fnc_cbaSettings.sqf]: `_settingName` is already a string judging by name.

File: addons/common/functions/fnc_parseList.sqf [hunk 3]
```diff
@@ -43,7 +43,7 @@ if (_checkNil) then {
     private _nilCheckedList = [];
 
     {
-        if !(isNil _x) then {
+        if (!isNil "_x") then {
```
Comment (commy2) [inline addons/common/functions/fnc_parseList.sqf]: broken

File: addons/arsenal/XEH_postInit.sqf [hunk 4]
```diff
@@ -24,7 +24,7 @@ GVAR(lastSearchTextLoadouts) = "";
 [QGVAR(loadoutUnshared), {
     params ["_contentPanelCtrl" , "_playerName", "_loadoutName"];
 
-    if (!(isNil QGVAR(currentLoadoutsTab)) && {GVAR(currentLoadoutsTab) == IDC_buttonSharedLoadouts}) then {
+    if ((!isNil QGVAR(currentLoadoutsTab)) && {GVAR(currentLoadoutsTab) == IDC_buttonSharedLoadouts}) then {
```
Comment (commy2) [inline addons/arsenal/XEH_postInit.sqf]: (!isNil QGVAR(currentLoadoutsTab))
->
!isNil QGVAR(currentLoadoutsTab)

**Comment by Whigital** [comment]
I see what you mean 🤦‍♂ . Will sort them out ....

**Comment by Whigital** [comment]
Think i caught most of the quotation wrong doings ....

**Comment by commy2** [comment]
lgtm

File: addons/tagging/functions/fnc_createTag.sqf [hunk 5]
```diff
@@ -46,7 +46,7 @@ if (_object getVariable [QGVAR(testVar), false]) then {
 
     // If the object already has tags attached, just add the new one to the list
     private _attachedTags = _object getVariable QGVAR(attachedTags);
-    if !(isNil  "_attachedTags ") exitWith {
+    if (!isNil  "_attachedTags ") exitWith {
```
Comment (mharis001) [inline addons/tagging/functions/fnc_createTag.sqf]:
```suggestion
    if (!isNil "_attachedTags") exitWith {
```

File: addons/arsenal/functions/fnc_onSelChangedLeft.sqf [hunk 6]
```diff
@@ -23,10 +23,10 @@ private _display = ctrlParent _control;
 private _item = [_control lnbData [_curSel, 0], _control lbData _curSel] select !(ctrlType _control == 102);
 
 private _weaponDefaultRightPanel = _display displayCtrl IDC_buttonOptic;
-private _selectCorrectPanelWeapon = [_weaponDefaultRightPanel, _display displayCtrl GVAR(currentRightPanel)] select (!(isNil QGVAR(currentRightPanel)) && {GVAR(currentRightPanel) in [RIGHT_PANEL_ACC_IDCS, IDC_buttonCurrentMag, IDC_buttonCurrentMag2]});
+private _selectCorrectPanelWeapon = [_weaponDefaultRightPanel, _display displayCtrl GVAR(currentRightPanel)] select ((!isNil QGVAR(currentRightPanel)) && {GVAR(currentRightPanel) in [RIGHT_PANEL_ACC_IDCS, IDC_buttonCurrentMag, IDC_buttonCurrentMag2]});
```
Comment (Whigital) [inline addons/arsenal/functions/fnc_onSelChangedLeft.sqf]: `(!isNil QGVAR(currentRightPanel))` -> `!isNil QGVAR(currentRightPanel)` ?

**Pull request merged**